hal/src/thumbv7em/clock/v2.rs

```diff
@@ -154,19 +153,21 @@ pub struct Tokens {
     /// Synchronous clocking domain clocks -- APB buses
     pub apbs: apb::ApbClks,
     /// Construction token for [`dpll::Dpll0`]
-    pub dpll0: dpll::DpllToken<dpll::marker::Dpll0>,
+    pub dpll0: dpll::DpllToken<dpll::DpllId0>,
```

Comment:
I think it make sense for `*Id` types to have `Id` suffix at the actual end. `Dpll0Id` > `DpllId0`. @vcchtjader agreed with me on that matter but maybe you have a different take on this.

Reply:
Yeah, I was leaning that way as well. The existing approach was the simplest to implement with `seq`. I'll have to figure out how to change it.